.github/ISSUE_TEMPLATE/refactoring.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Alternatively you could we could just put this in https://github.com/opendcs/.github/tree/main/.github/ISSUE_TEMPLATE and it'll get picked up.

Reply:
odd, they did for me, except refactoring which wasn't there.

Reply:
whelp, I must have been looking at my fork or something because I see it now.

Reply:
removed the templates since I was just copying the existing ones. if we want to customize in the future we can.

Reply:
created PR: https://github.com/opendcs/.github/pull/2 to add the refactoring template for the organization